server/api/order.js

```diff
@@ -0,0 +1,41 @@
+const router = require('express').Router();
+const { Order, LineItem } = require('../db/models');
+
+module.exports = router;
+
+//GET ALL ORDERS
+router.get('/', (req, res, next) => {
+    Order.findAll()
+        .then(orders => res.json(orders)) 
+        .catch(next);
+});
+
+//POST ORDER NEED TO ADD DEFAULT VALUE OF STATUS AS OPEN
```

Comment:
Default value should be 'shopping cart' or something like that since we'll store the shopping cart data in our database